docs: change IPC window_type value

ctemplin opened the pull request:
from "undefined" to "unknown" as returned by the `get_tree` command.

ctemplin committed:
docs: change IPC window_type value

from "undefined" to "unknown" as returned by the `get_tree` command.

Airblader reviewed (changes requested):
Thank you so much for the PR to keep the docs up to date! `undefined` is actually _also_ correct here as that is returned when a container has no window; however, this is a true `undefined`, not the string `"undefined"`, which is a bit misleading.

Of course you are correct that `"unknown"` also exists. Could you change this into mentioning that additionally instead of removing `undefined`? Feel free to change the wording to clarify this difference, too, if you want!

ctemplin committed:
docs: IPC window_type undefined

Re-adding `undefined` with code formatting to distinguish it from string values.

ctemplin commented:
Thanks for the clarification. I've re-added `undefined` and made what I think is the minimum change required to convey the distinction.

Airblader reviewed (approved):
Thanks again! :-)
